### DISCUSSION TITLE
wiki:3.0.0-alpha.363 username and password

### DISCUSSION THREAD

**Discussion by fmacedovectrading** [requarks/wiki#7292] [Help / Questions]
I've downloaded the last container image via docker pull and started the container. I can access via web but it shows a initial screen asking for email and password, but it did not had the chance to configure it via the 3000 port.  Whats the default email/password to login?

**Comment by fnafox** [comment]
Email: admin@example.com
    Password: 12345678

src: https://github.com/requarks/wiki/blob/vega/README.md
